### ISSUE TITLE
IDE Freezing

### ISSUE THREAD

**Issue by PeterKaa1** [radimitrov/CSharpShellApp#336]
When I started a new .NET MAUI project and created a new keystore, I saved the project. However, when I attempted to go back to the keystore (signing) menu, the IDE froze and did not load that menu. I tried restarting my device and reinstalling both the IDE and the plugin app, but the issue still persists.

**Comment by radimitrov** [comment]
Yes, is is fixed now. Unexpectedly, it was the keystore library password decryption method hanging. I had to replace it so keystore & alias passwords will need to be reentered when the app is updated within a few days.
